gunicorn/config.py

```diff
@@ -69,6 +69,12 @@ def set(self, name, value):
             raise AttributeError("No configuration setting for: %s" % name)
         self.settings[name].set(value)
 
+    def get_gunicorn_env_var(self):
```

Comment:
I'm not a native English speaker, but perhaps we can rename it to something like `get_cmd_args_from_env`? `get_gunicorn_env_var` sounds too generic to me. Ping @tilgovi :)